portal/static/js/profile.js

```diff
@@ -614,7 +614,7 @@
                 if (targetSection.length > 0) {
                     var loadingElement = targetSection.find(".profile-item-loader");
                     loadingElement.show();
-                    this.modules.tnthAjax.getOptionalCoreData(self.subjectId, {useWorker: true}, function(data) {
+                    this.modules.tnthAjax.getOptionalCoreData(self.subjectId, {useWorker: true, cache: true}, function(data) { //cache this request as change is rare if ever for optional data
```

Comment:
I think we might be able to cache more requests without any effort if the default were to always cache, unless we know for sure we need fresh data

Reply:
@ivan-c ivan, i think more often than not we want fresh data especially in profile and initial queries page - will set default to always cache if I know for sure it is safe to do so after more rounds of testing.